src/jloptions.c

```diff
@@ -154,11 +155,18 @@ static const char opts[]  =
     // instrumentation options
     " --code-coverage[={none*|user|all}]\n"
     "                            Count executions of source lines (omitting setting is equivalent to `user`)\n"
+    " --code-coverage=@/abs/path\n"
```

Comment:
I don't think any other file input requires an absolute path, couldn't the path be absolutified instead?

Reply:
Good point! I was unaware how the paths were absolutified. 

The nice thing now is that a simple `--code-coverage=@` will track the current directory